.travis.yml

```diff
@@ -16,7 +16,7 @@ before_script:
 script:
   - go get -d -v
```

Comment:
I think we don't need to run `go get` since we have the vendor dir on git.

Reply:
Agreed we can remove that line now.